### PR TITLE
Added changes for Test Email option

### PR DESCRIPTION
**Pull request by catalinaadam**
Added description for Test Email Configuration and improved the current structure of the "Set up a communication channel" topic.

### CODE REVIEW

**Comment by rnovikovP** [comment]
https://github.com/percona/pmm-doc/pull/668/commits/27dcaf7372691dbee75725126ff635f24f685238  
 LGTM
but I'm  🤔  about other commits. They need to be verified, as  they can add old text because of branch changes

**Review by rasika-chivate — APPROVED** [review]

**Comment by fabio-silva** [comment]
> [27dcaf7](https://github.com/percona/pmm-doc/commit/27dcaf7372691dbee75725126ff635f24f685238) LGTM but I'm 🤔 about other commits. They need to be verified, as they can add old text because of branch changes

Yeah, this branch was based on main, we just changed that to 2.26.0, all good now 👍

File: docs/release-notes/2.26.0.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# Percona Monitoring and Management 2.21.0
```
Comment (fabio-silva) [inline docs/release-notes/2.26.0.md:1]: 2.26.0 ?

File: docs/release-notes/2.26.0.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+<table class="docutils field-list" frame="void" rules="none">
+  <colgroup>
+    <col class="field-name">
+    <col class="field-body">
+  </colgroup>
+  <tbody valign="top">
+    <tr class="field-odd field">
+      <th class="field-name">Date:</th>
+      <td class="field-body">August 26, 2021</td>
```
Comment (fabio-silva) [inline docs/release-notes/2.26.0.md:11]: should we change this date?

**Review by fabio-silva — CHANGES_REQUESTED** [review]

**Review by fabio-silva — APPROVED** [review]
🚀 🚀

**Review by rnovikovP — APPROVED** [review]

**Pull request merged**